server/controllers/rideRequestController.js

```diff
@@ -78,4 +87,62 @@ export default class RideRequest {
         });
       });
   }
+
+  /**
+   * accept or reject a ride request
+   *
+   * @static
+   * @param {*} req
+   * @param {*} res
+   * @memberof RideRequest
+   */
+  static respondToRequests(req, res) {
+    const { status } = req.body;
+    const { requestId, rideId } = req.params;
+    const { userId } = req;
+    // check if the user is the one who created the ride offer
+    pool
+      .query('SELECT * FROM rides WHERE id = $1', [rideId])
+      .then((result) => {
```

Comment:
you can use object destructuring here and also rename `rows` to rides`

Reply:
Ok. Thanks. 
I'll complete this in my next commit.